### DISCUSSION TITLE
How to reset an entity like sensor.ignored_issues or sensor.issues?

### DISCUSSION THREAD

**Discussion by elishaip** [frenck/spook#751] [Q&A] [answered]
How to reset an entities like sensor.ignored_issues or sensor.issues to 0?

Thanks you!

**Comment by frenck** [comment, marked as answer]
You don't? As in, they reflect the number of issues in your repairs dashboard in Home Assistant.

If you resolve repairs, the number will go down 🤷 

../Frenck

**Reply by elishaip** [reply]
The **sensor.active_issues** entity does go down but the other entities don't reset or go down. See the screenshot.

By the way, the very high number was created because I did a wrong automation that produced a lot of correction messages and therefore I would like to reset the entities.

![2024-05-09_212105](https://github.com/frenck/spook/assets/89676547/229669b0-f20d-4aff-93ae-b9a800c2a936)
